packages/graphql/lib/src/core/query_manager.dart

```diff
@@ -36,12 +36,6 @@ class QueryManager {
   Map<String, ObservableQuery> queries = <String, ObservableQuery>{};
 
   ObservableQuery watchQuery(WatchQueryOptions options) {
-    if (options.document == null) {
```

Comment:
Can we safely drop this?

Reply:
yes, because of `assert(document != null)` in `RawOperationData`